test/api_test.js

```diff
@@ -21,14 +21,14 @@ suite('API', () => {
     await namespaces.removeTable();
   });
 
-  test('ping', () => {
+  test.skip('ping', () => {
```

Comment:
I rebased master onto this branch, but it looks like I cannot find any object which has a `ping` method on it, hence this test is still failing for me :(

Which file holds the JS class defining the `ping` method?